src/remote/utils.go

```diff
@@ -35,6 +37,24 @@ import (
 // xattrName is the name we use to record attributes on files.
 const xattrName = "user.plz_hash_remote"
 
+func (c *Client) pushTreeDigestDownloadError() {
+	if c.state.Config.Remote.PrometheusGatewayURL == "" {
+		log.Infof("No Prometheus pushgateway URL to push Digest Download error to")
+		return
+	}
+	downloadErrorCounter := prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "please",
+		Name:      "tree_digest_download_eof_error",
+	})
+	if err := push.New(
+		c.state.Config.Remote.PrometheusGatewayURL, "tree_digest_download",
+	).Collector(downloadErrorCounter).Push(); err != nil {
+		log.Warningf("Error pushing to Prometheus pushgateway: %s", err)
+		return
+	}
+	log.Infof("Incremented error counter via pushgateway")
```

Comment:
Again not sure this is info worthy. I feel info is more about letting you know what state the system is in mostly during setup e.g. "starting connection pool",  "listening on port 8080" and stuff. We don't want to log for business as usual things.